src/components/Layout/Editor/Editor.tsx

```diff
@@ -12,12 +17,30 @@ type EditorType = {
 
 const Editor = ({ onChange, initialValue }: EditorType) => {
   const { colorMode } = useColorMode()
+  const [language, setLanguage] = useState('en-US')
   const editorRef = useRef(null)
```

Comment:
Type the ref, so you can avoid using `any` type when you call it
```suggestion
  const editorRef = React.useRef<ToastUIEditor>(null);
```

Reply:
@anubra266 right. I'll be more careful on that. Thanks!